provider/resources.go

```diff
@@ -98,28 +98,27 @@ func Provider() tfbridge.ProviderInfo {
 			},
 		},
 		Resources: map[string]*tfbridge.ResourceInfo{
-			"azuread_application":                         {Tok: makeResource(mainMod, "Application")},
-			"azuread_application_password":                {Tok: makeResource(mainMod, "ApplicationPassword")},
-			"azuread_group":                               {Tok: makeResource(mainMod, "Group")},
-			"azuread_service_principal":                   {Tok: makeResource(mainMod, "ServicePrincipal")},
-			"azuread_service_principal_password":          {Tok: makeResource(mainMod, "ServicePrincipalPassword")},
-			"azuread_user":                                {Tok: makeResource(mainMod, "User")},
-			"azuread_group_member":                        {Tok: makeResource(mainMod, "GroupMember")},
-			"azuread_application_certificate":             {Tok: makeResource(mainMod, "ApplicationCertificate")},
-			"azuread_application_app_role":                {Tok: makeResource(mainMod, "ApplicationAppRole")},
```

Comment:
Do we need to changelog this removal too?

Reply:
Yes! I will fix that up :) Thank you! Great catch